[malwarebytes-254] Update README.md (#13939)
demisto/content: Packs/Malwarebytes/README.md

> Malwarebytes Cloud integrates with Demisto for interactions with Incident Response, Endpoint Protection, and Endpoint Protection and Response products. Malwarebytes makes endpoints resilient through adaptive cyber protection, active threat response and orchestrated endpoint control that quickly restores workforce productivity when a compromise occurs. Our adaptive cyber protection predicts and detects attacks with multi-layer detection across the kill chain. We enable active threat response where machine learning is actionable and automated allowing for full recovery when a compromise occurs. We orchestrate command and control across siloed IT and Security organizations simplifying security management and making response effective. Malwarebytes makes endpoints resilient, so that you can protect, remediate, and regain control of your digital business.  

Reviewer: ```suggestion
Malwarebytes Cloud integrates with Cortex XSOAR for interactions with Incident Response, Endpoint Protection, and Endpoint Protection and Response products. Malwarebytes makes endpoints resilient through adaptive cyber protection, active threat response and orchestrated endpoint control that quickly restores workforce productivity when a compromise occurs. Our adaptive cyber protection predicts and detects attacks with multi-layer detection across the kill chain. We enable active threat response where machine learning is actionable and automated allowing for full recovery when a compromise occurs. We orchestrate command and control across siloed IT and Security organizations simplifying security management and making response effective. Malwarebytes makes endpoints resilient, so that you can protect, remediate, and regain control of your digital business.  

```

##### What does this pack do?
The playbooks included in this pack help you to Scan & Remediate endpoints and Isolate endpoints. They also help with the following use cases:
- Trigger Malwarebytes Scans and Remediation.

> - Trigger Malwarebytes EDR Advanced Capabilities

Reviewer: ```suggestion
- Trigger Malwarebytes EDR Advanced Capabilities.

```

- Create Demisto incidents based on threats detected by Malwarebytes.

Also, customers can find the following helpful information:

> - Product link - https://cloud.malwarebytes.com/

Reviewer: ```suggestion
- [Product Information](https://cloud.malwarebytes.com/)

```

> - Support documentation link - https://support.malwarebytes.com/hc/en-us/sections/360008537673-Palo-Alto-Networks-Cortex-XSOAR

Reviewer: ```suggestion
- [Support Documentation](https://support.malwarebytes.com/hc/en-us/sections/360008537673-Palo-Alto-Networks-Cortex-XSOAR)
```
